lessons/Early-Math/associative-property/langs/en-us/index.adoc

```diff
@@ -0,0 +1,123 @@
+= The Associative Property
+
+@description{This lesson introduces students to the Associative Property by way of Circles of Evaluation.}
+
+@lesson-prereqs{commutative-property}
+
+@keywords{Associative Property}
+
+[@lesson-intro-table]
+|===
+
+| Lesson Goals
+| Students will be able to...
+
+* Recognize that, no matter how we group numbers being either added or multiplied, the end result will always be the same.
+* Use the Associative Property to simplify expressions and make computation easier.
+
+| Student-facing Lesson Goals
+|
+
+* Let's explore the Associative Property via Circles of Evaluation.
+
+| Materials
+|[.materials-links]
+@material-links
+
+| Key Points For The Facilitator
+|
+* The Associative Property - like other laws of arithmetic - is more meaningful and memorable when students view it as a powerful tool to help them to do math efficiently.
+* Students may need support in parsing expressions that include both an operation that _does_ support Associativity along side an operation where Associativity does _not_ apply.
+|===
+
```

Comment:
Noting that it might make more sense for these definitions to be fully parallel in wording.
<img width="835" alt="Screen Shot 2023-01-27 at 12 36 35 PM" src="https://user-images.githubusercontent.com/66691020/215167696-7287546b-cbd6-400f-87ea-db5762eefe83.png">